tests/inputFields.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach( async({page}) => {
+  await page.goto('/')
+  //1. Select the PET TYPES menu item in the navigation bar
+  await page.getByText('Pet Types').click()
+  //2. Add assertion of the "Pet Types" text displayed above the table with the list of pet types
+  expect(await page.locator('h2').getByText('Pet Types').isVisible()).toBeTruthy()
+})
+
+test('Update pet type', async ({page}) => {
+  test.slow()
+  //3. Click on "Edit" button for the "cat" pet type
+  //assumes cat is always first
+  await page.getByText('Edit').first().click()
+
+  //4. Add assertion of the "Edit Pet Type" text displayed
+  expect(await page.getByText('Edit Pet Type').isVisible()).toBeTruthy()
+
+  //5. Change the pet type name from "cat" to "rabbit" and click "Update" button
+  const textField = page.locator('#name')
+  await textField.click()
+  await textField.clear()
+  await textField.pressSequentially('rabbit')
```

Comment:
Was there any reason why you wanted to use` pressSequentially()` method?
This method is used when it's needed to simulate a key strokes. If all you need is to fill the input field, better to use method `fill()`